packages/type-jsonrpc/src/create/method.ts

```diff
@@ -4,7 +4,7 @@
 
 import { RpcMethodOpt, RpcMethod } from '../types';
 
-import isUndefined from '@polkadot/util/is/undefined';
+import { isUndefined } from '@polkadot/util';
```

Comment:
Relative imports before absolute imports is uncommon.

Reply:
It is a types split above - and yes, with those with the new types things have become _very_ muddy